experimental/Schemes/Auxiliary.jl

```diff
@@ -40,6 +41,18 @@ function (f_star::UniversalPullbackSymbol)(M::Any)
   return pullback(f_star.f, M)
 end
 
+# We can do the same thing with pushforward and other symbols.
+struct UniversalPushforwardSymbol{MorphismType}
+  f::MorphismType
+end
+
+# The following refers a call of f_star(M) to a bivariate pullback-method.
```

Comment:
please adjust this copy of the pullback comment to the setting of pushforward: f_lower_star, pushforward